@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -331,7 +331,8 @@ const fillTechniqueDescriptor = mergeTechniqueDescriptor<FillTechnique>(
             color: AttrScope.TechniqueRendering,
             opacity: AttrScope.TechniqueRendering,
             transparent: AttrScope.TechniqueRendering,
-            lineWidth: AttrScope.TechniqueRendering
+            lineWidth: AttrScope.TechniqueRendering,
+            clipping: AttrScope.TechniqueRendering
```

Comment:
This should be `AttrScope.TechniqueGeometry` because _NOW_ there is no way to re-evaluate this property on each frame.

Reply:
I fixed it to use getPropertyValue... so this can stay the same if I understand correct?

Reply:
Yep.